cogs/reward.py

```diff
@@ -43,89 +45,118 @@ def convert_to_seconds(s):
-
-            if totalTime == 0:
-                await channel.send(content="You may have formatted the time incorrectly or calculated for 0. Try again with the correct format." + rewardCommand)
-                return
 
             cp = ((totalTime) // 1800) / 2
             tier = tier.lower()
-            t = 0
+            tierNum = 0
             if tier == 'junior':
-              t = 0
+              tierNum = 0
             elif tier == "journey":
-              t = 1
+              tierNum = 1
             elif tier == "elite":
-              t = 2
+              tierNum = 2
             elif tier == "true":
-              t = 3
+              tierNum = 3
             elif tier == "ascended":
-              t = 4
+              tierNum = 4
             
-            gp = cp* tier_reward_dictionary[t][0]
-            tp = cp* tier_reward_dictionary[t][1]
+            gp = cp* tier_reward_dictionary[tierNum][0]
+            tp = cp* tier_reward_dictionary[tierNum][1]
 
             treasureArray = [cp, tp, gp]
             durationString = timeConversion(totalTime)
             treasureString = f"{treasureArray[0]} CP, {treasureArray[1]} TP, and {treasureArray[2]} GP"
             await channel.send(content=f"A {durationString} game would give a **{tierName}** Friend\n{treasureString}")
             return
-        elif characterPresent == True:
-            l = list((re.findall('.*?[hm]', lowerTimeString)))
-            totalTime = 0
-            for timeItem in l:
-                totalTime += convert_to_seconds(timeItem)
+        else:  # Calculates rewards and output if a character was given.
+                        
 
-            if totalTime == 0:
-                await channel.send(content="You may have formatted the time incorrectly or calculated for 0. Try again with the correct format." + rewardCommand)
-                return
-            
-            t = 0
             if charDict["Level"] < 5:
-                t = 0
+                tierNum = 0
                 tierInitial = 'T1 TP'
                 tierNext = 'T2 TP'
+                tierNext1 = 'T3 TP'
+                tierNext2 = 'T4 TP'
+                tierNext3 = 'T5 TP'
             elif charDict["Level"] < 11:
-                t = 1
+                tierNum = 1
                 tierInitial = 'T2 TP'
                 tierNext = 'T3 TP'
+                tierNext1 = 'T4 TP'
+                tierNext2 = 'T5 TP'
             elif charDict["Level"] < 17:
-                t = 2
+                tierNum = 2
                 tierInitial = 'T3 TP'
                 tierNext = 'T4 TP'
+                tierNext1 = 'T5 TP'
             elif charDict["Level"] < 20:
-                t = 3
+                tierNum = 3
                 tierInitial = 'T4 TP'
                 tierNext = 'T5 TP'
-            elif charDict["Level"] == 20:
-                t = 4
+            else:
+                tierNum = 4
                 tierInitial = 'T5 TP'
```

Comment:
All the tierNextX values are already contained within the return value of `calculateTreasure()`
They are not needed and just clutter.